spec/RuleSpec.js

```diff
@@ -0,0 +1,183 @@
+describe('Testing class Rule',function(){
+	describe("Testing isRequired",function(){
+		it("Testing isRequired - null", function(){
+			let dataName = {
+				name: null
+			},
+			dataNameRule = {
+				name: new Rule().isRequired()
+			};
+			expect(dataNameRule.name.validateRule(dataName.name)[0]).toBe('Ничего не введено');
+		})
+		it("Testing isRequired - all good(name is Robin)",function(){
+			let dataName = {
+				name: "Robin"
+			},
+			dataNameRule = {
+				name: new Rule().isRequired()
+			};
+			expect(dataNameRule.name.validateRule(dataName.name)[0]).toBe();
```

Comment:
Separate Act from Assert part, so that it will conform AAA pattern and improve readability:

let result = dateNameRule.name.validateRule(dateName.name);

expect(result).toBe()